ui/tests/acceptance/secrets/backend/ssh/role-test.js

```diff
@@ -3,110 +3,221 @@
+import enablePage from 'vault/tests/pages/settings/mount-secret-backend';
+import listPage from 'vault/tests/pages/secrets/backend/list';
 import editPage from 'vault/tests/pages/secrets/backend/ssh/edit-role';
 import showPage from 'vault/tests/pages/secrets/backend/ssh/show';
 import generatePage from 'vault/tests/pages/secrets/backend/ssh/generate-otp';
-import listPage from 'vault/tests/pages/secrets/backend/list';
-import enablePage from 'vault/tests/pages/settings/mount-secret-backend';
-import authPage from 'vault/tests/pages/auth';
+import { runCmd } from 'vault/tests/helpers/commands';
+import { GENERAL } from 'vault/tests/helpers/general-selectors';
+import { SECRET_ENGINE_SELECTORS as SES } from 'vault/tests/helpers/secret-engine/secret-engine-selectors';
+
+// There is duplication within this test suite. The duplication occurred because two test suites both testing ssh roles were merged.
+// refactoring the tests to remove the duplication would be a good next step as well as removing the tests/pages.
 
-module('Acceptance | secrets/ssh', function (hooks) {
+module('Acceptance | ssh | roles', function (hooks) {
   setupApplicationTest(hooks);
 
   hooks.beforeEach(function () {
     this.uid = uuidv4();
     return authPage.login();
   });
 
-  const mountAndNav = async (uid) => {
-    const path = `ssh-${uid}`;
-    await enablePage.enable('ssh', path);
-    await settled();
-    await editPage.visitRoot({ backend: path });
-    await settled();
-    return path;
-  };
-
-  test('it creates a role and redirects', async function (assert) {
-    assert.expect(5);
-    const path = await mountAndNav(this.uid);
-    await editPage.createOTPRole('role');
-    await settled();
-    assert.strictEqual(
-      currentRouteName(),
-      'vault.cluster.secrets.backend.show',
-      'redirects to the show page'
-    );
-    assert.ok(showPage.generateIsPresent, 'shows the generate button');
-
-    await showPage.visit({ backend: path, id: 'role' });
-    await settled();
-    await showPage.generate();
-    await settled();
-    assert.strictEqual(
-      currentRouteName(),
-      'vault.cluster.secrets.backend.credentials',
-      'navs to the credentials page'
-    );
-
-    await listPage.visitRoot({ backend: path });
-    await settled();
-    assert.strictEqual(listPage.secrets.length, 1, 'shows role in the list');
-    const secret = listPage.secrets.objectAt(0);
-    await secret.menuToggle();
-    assert.dom('.hds-dropdown li').exists({ count: 5 }, 'Renders 5 popup menu items');
-  });
+  const PUB_KEY = `ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQCn9p5dHNr4aU4R2W7ln+efzO5N2Cdv/SXk6zbCcvhWcblWMjkXf802B0PbKvf6cJIzM/Xalb3qz1cK+UUjCSEAQWefk6YmfzbOikfc5EHaSKUqDdE+HlsGPvl42rjCr28qYfuYh031YfwEQGEAIEypo7OyAj+38NLbHAQxDxuaReee1YCOV5rqWGtEgl2VtP5kG+QEBza4ZfeglS85f/GGTvZC4Jq1GX+wgmFxIPnd6/mUXa4ecoR0QMfOAzzvPm4ajcNCQORfHLQKAcmiBYMiyQJoU+fYpi9CJGT1jWTmR99yBkrSg6yitI2qqXyrpwAbhNGrM0Fw0WpWxh66N9Xp meirish@Macintosh-3.local`;
```

Comment:
This was here before, not added. Might help to compare to current [ssh-test file](https://github.com/hashicorp/vault/blob/main/ui/tests/acceptance/secrets/backend/ssh/ssh-test.js) as the diff is a little hard to parse.